supp2u/src/components/Navigation/Navigation.js

```diff
@@ -16,15 +18,32 @@ export default function Navigation(props) {
         }
     };
 
+    let renderUserType = () => {
```

Comment:
This looks good Laryna. You _could_ maybe throw that `.getItem` logic into a couple of variables to simplify your if-else block, but the div this is getting called from is way cleaner now 👍 
```
let renderUserType = () => {
      const isCustomer = localStorage.getItem('customer_id') && localStorage.getItem('user_id');
      const isOwner = localStorage.getItem('business_id') && localStorage.getItem('user_id');
       
       if (isCustomer) return <CustomerNav />
       if (isBusiness) return <BusinessNav />
       return <NavLink className="Link" to="/"></NavLink>
}
```

Reply:
thanks!